www/include/options/accessLists/reloadACL/reloadACL.php

```diff
@@ -103,14 +119,20 @@
         $session_data[$cpt]["user_id"] = $r["user_id"];
         $session_data[$cpt]["user_alias"] = $rU["contact_name"];
         $session_data[$cpt]["admin"] = $rU["contact_admin"];
-        
-        $resCP = $pearDB->query("SELECT topology_name, topology_page, topology_url_opt FROM topology WHERE topology_page = '".$r["current_page"]."'");
-        $rCP = $resCP->fetchRow();
+
+        $resCP = $pearDB->prepare(
```

Comment:
I think you can place this subquery directly in the first request to relieve the database (SELECT DISTINCT * FROM session)

Reply:
no I can't as it uses the results of the second query (is admin) to display on wich page are currently each users

Reply:
and if the topology_page result is 0 or something not inserted properly in the table, the result will be inconsistent or empty